cloud/disk_manager/test/acceptance/test_runner/sync_acceptance_test_runner.py

```diff
@@ -19,8 +21,37 @@ class SyncTestBinaryExecutor(BaseTestBinaryExecutor):
     _entity_suffix = 'sync'
 
 
+class SyncTestCleaner(BaseResourceCleaner):
+    def __init__(self, ycp: YcpWrapper, args: argparse.Namespace):
+        super(SyncTestCleaner, self).__init__(ycp, args)
+        test_type = args.test_type
+        disk_size = size_prettifier(args.disk_size * (1024 ** 3)).lower()
+        disk_blocksize = size_prettifier(args.disk_blocksize).lower()
+        disk_name_string = (
+            fr'^acceptance-test-{test_type}-'
+            fr'{disk_size}-'
+            fr'{disk_blocksize}-[0-9]+'
+        )
+        disk_name_pattern = re.compile(fr'{disk_name_string}$')
+        secondary_disk_name_pattern = re.compile(
+            fr'{disk_name_string}-from-snapshot$',
+        )
+        self._entity_ttls = self._entity_ttls | {
```

Comment:
Do we really need separate TTLs for different entities?
Lets use one value for all of them?

Reply:
Yep, VM's should not live for a long time, since they are re-created for each test.